Dockerfile.cpu

```diff
@@ -22,7 +22,7 @@ ENV LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libtcmalloc_minimal.so.4:/usr/local/li
 
 RUN echo 'ulimit -c 0' >> ~/.bashrc
 
-RUN pip install intel_extension_for_pytorch==2.5.0
+RUN pip install intel_extension_for_pytorch==2.5.0 # FIXME: should this be a dependency in requirements-cpu.txt?
```

Comment:
Does anybody know if there's a specific reason on why this was added here instead of being added to `requirements-cpu.txt`?

One potential issue could be that `requirements-cpu.txt` also is used by ppc and arm Dockerfiles, although this could fixed by adding 

```
intel_extension_for_pytorch==2.5.0; platform_machine == "x86_64"
``` 

This does not solve the issue when using AMD processors, but this is currently when building Dockerfile.cpu

Reply:
yeah, it'd be nicer in "requirements".